internal/rbd/rbd_util.go

```diff
@@ -480,15 +466,12 @@ func genSnapFromSnapID(ctx context.Context, rbdSnap *rbdSnapshot, snapshotID str
 			return err
 		}
 	}
-
-	err = updateSnapWithImageInfo(ctx, rbdSnap, cr)
-
-	return err
+	return nil
 }
 
 // genVolFromVolID generates a rbdVolume structure from the provided identifier, updating
 // the structure with elements from on-disk image metadata as well
-func genVolFromVolID(ctx context.Context, volumeID string, cr *util.Credentials, secrets map[string]string) (*rbdVolume, error) {
+func genVolFromVolID(ctx context.Context, volumeID string, cleanupVol bool, cr *util.Credentials, secrets map[string]string) (*rbdVolume, error) {
```

Comment:
I was wondering about the new boolean that was passed to this function. However, `cleanupVol` does not help me at all. Maybe document its intention?